include/wil/common.h

```diff
@@ -577,13 +596,17 @@ doing it with global function pointers and header initialization allows a runtim
     } \
     __pragma(comment(linker, "/INCLUDE:_g_header_init_" #name))

```

Comment:
What about this for `_M_IX86`?

Reply:
Great i've closed mine as yours looks cleaner